### PR TITLE
Limit label list height on edit issue page

### PR DESCRIPTION
**Pull request by ghost**
Before was no limit:

Screenshot_2020-02-10_23-15-19

After:

Screenshot_2020-02-10_23-10-48

### CODE REVIEW

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -158,6 +158,8 @@
         }
 
         &.labels {
+            max-height: 300px;
```
Comment (guillep2k) [inline web_src/less/_repository.less:161]: Maybe we should use [`vh`](https://www.w3schools.com/cssref/css_units.asp) for this? (e.g. `max-size: 30vh`) (I'm not familiar with any limitations or caveats that may come with it, [backwards compatibility](https://caniuse.com/#feat=viewport-units), etc.).

Comment (silverwind) [reply web_src/less/_repository.less:161]: `max-height: calc(50vh - anchorheight - margin)` should be safe in all cases, assuming the positioning script is smart enough to anchor upwards when the button is on the end of the vertical viewport.

Comment (proton-ab) [reply web_src/less/_repository.less:161]: I believe the author is reproducing behavior from `.repository.new.issue .comment.form .metas .filter.menu` which sets 300px.

**Comment by jamesorlakin** [comment]
This also happens on the assignees on 1.11.0. I assume it's the same behaviour for labels but we don't have many... I think this should be backported, I'm happy to make a PR.

**Comment by jaqra** [comment]
as @proton-ab said i copied this rule from `New Issue` page

@lafriks marked as resolved. so i need to do anything?

@jamesorlakin ofcourse you can make a backport after merge :)

**Review by 6543 — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]
I think it's OK to maintain the 300px value for consistency, but perhaps we should change them all to a relative height in a future PR.

This one LGTM 👍

**Review by sapk — APPROVED** [review]

**Comment by jamesorlakin** [comment]
Having had a closer look while creating a backport, I'm not sure this will also fix a long assignments list as well. The extra CSS property is only assigned to the label selector, whereas the underlined selector is common between the two (`_repository.less:91`) and I feel should be changed instead.

image
_(Names redacted because this is my work's 1.11 instance)_

**Comment by jaqra** [comment]
@jamesorlakin can you send a pr?

**Comment by jamesorlakin** [comment]
Sure, I'll make a PR against master and then experiment with backporting it 👍

**Pull request merged**